frontend/src/components/SortableTaskList.module.scss

```diff
@@ -29,19 +29,14 @@
   @extend .searchBarContainer;
   @extend .layout-column;
   background-color: $COLOR_BLACK10;
-  padding: 10px 0px;
+  padding: 10px 3px;
   gap: 10px;
   position: sticky;
   top: 0;
 }
 
-.taskName {
-  @extend .taskNameContainer;
-  width: 175px;
-}
-
 .measureTaskName {
-  @extend .taskName;
+  @extend .taskNameContainer;
   visibility: hidden;
 }
```

Comment:
Just removing this unfortunately breaks the measuring. 
The measure div should *always* have the *exactly same width* as what the name will have in the real list item. Otherwise there will be cases where the measured text uses different number of rows than the actual text.

These available widths are different for the milestones and for the unordered-list, 208.5 and 175.75 pixels respectively as I checked. Both happen to be narrower than the 210 pixels that is given for the measuring div.

I think the measuring is overall bit too fragile at the moment, breaking too easily on style changes and the breaking being too easy to not notice.

I think using hard coded widths and verifying that those have actually the full width in the dom, is the most reasonable without larger refactoring. Although a better way to do this would be great...